src/lib/evil/evil_main.c

```diff
@@ -44,21 +44,8 @@ evil_init(void)
 
    _evil_time_freq = freq.QuadPart;
 
-   /* be sure that second + 1 != 0 */
-   while (second == 59)
-     {
-        GetSystemTime(&st);
-        second = st.wSecond;
-     }
-
-   /* retrieve the tick corresponding to the time we retrieve above */
-   while (1)
-     {
-        GetSystemTime(&st);
-        QueryPerformanceCounter(&count);
-        if (st.wSecond == second + 1)
-          break;
-     }
```

Comment:
@felipealmeida could you shed some light on what this did and why it's no longer needed?

Reply:
Apparently this was used for an old implementation of `gettimeofday` that was abandoned. And this change is safe.